Kitodo-API/src/main/java/org/kitodo/api/dataformat/Workpiece.java

```diff
@@ -0,0 +1,176 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.dataformat;
+
+import java.util.ArrayList;
+import java.util.GregorianCalendar;
+import java.util.LinkedList;
+import java.util.List;
+
+import org.kitodo.api.Metadata;
+
+/**
+ * The administrative structure of the product of an element that passes through
+ * a Production workflow.
+ * 
+ * <p>
+ * A {@code Workpiece} has two essential characteristics: {@link MediaUnit}s and
+ * an outline {@link Structure}. {@code MediaUnit}s are the types of every
+ * single digital medium on a conceptual level, such as the individual pages of
+ * a book. Each {@code MediaUnit} can be in different {@link MediaVariant}s (for
+ * example, in different resolutions or file formats). Each {@code MediaVariant}
+ * of a {@code MediaUnit} resides in a place described by an URI.
+ * 
+ * <p>
+ * The {@code Structure} is a tree structure that can be finely subdivided, e.g.
+ * a book, in which the chapters, in it individual elements such as tables or
+ * figures. Each outline level points to the {@code MediaUnit}s that belong to
+ * it via {@link View}s. Currently, a {@code View} always contains exactly one
+ * {@code MediaUnit} unit, here a simple expandability is provided, so that in a
+ * future version excerpts from {@code MediaUnit}s can be described. Each
+ * outline level can be described with any {@link Metadata}.
+ */
+public class Workpiece {
+    /**
+     * The time this file was first created.
+     */
+    private GregorianCalendar creationDate = new GregorianCalendar();
+
+    /**
+     * The processing history.
+     */
+    private List<ProcessingNote> editHistory = new ArrayList<>();
+
+    /**
+     * The identifier of the workpiece.
+     */
+    private String id;
+
+    /**
+     * The media units that belong to this workpiece. The order of this
+     * collection is meaningful, but only describes the order in which the media
+     * units are displayed on the workstation of the compiler.
+     */
+    private List<MediaUnit> mediaUnits = new LinkedList<>();
+
+    /**
+     * The logical structure.
+     */
+    private Structure structure = new Structure();
+
+    /**
+     * Returns the creation date of the workpiece.
+     * 
+     * @return the creation date
+     */
+    public GregorianCalendar getCreationDate() {
+        return creationDate;
+    }
+
+    /**
+     * Returns the edit history. The head of each METS file has space for
```

Comment:
please cut down javadoc to 
Returns edit history.

If you really want to add information on the field, add it to the javadoc of the field, but I would rather have it clean.